common/views/components/Picture/Picture.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import type {Picture as PictureProps} from '../../../model/picture';
+import Tasl from '../Tasl/Tasl';
+import {imageSizes} from '../../../utils/image-sizes';
+import {convertImageUri} from '../../../utils/convert-image-uri';
+
+type Props = {|
+  images: PictureProps[],
+  extraClasses?: string
+|};
+
+const Picture = ({ images, extraClasses }: Props) => {
+  const lastImage = images[images.length - 1];
```

Comment:
Would at make any more/less sense to use `map`s index parameter (`map(item, index)`) and test if that equals images.length?

I guess this is more declarative.